core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -315,18 +375,15 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.editTopic = async function(topicName) {
-    await waitFor.elementToBeClickable(
-      topicsTabButton, 'Unable to click on topics tab.');
-    var topicElements = await _getTopicElements(topicName);
-    if (topicElements.length === 0) {
-      throw new Error('Could not find topic tile with name ' + topicName);
-    }
-    await waitFor.elementToBeClickable(
-      topicElements[0], 'Unable to click on topic: ' + topicName);
-    await topicElements[0].click();
-    await waitFor.pageToFullyLoad();
-    await waitFor.invisibilityOf(
-      topicElements[0], 'Topic editor page takes too long to load.');
+    await waitFor.visibilityOf(topicsTable,
+      'Topic table taking too long to appear.');
+
+    await Promise.all(await topicNames.map(async(topic, index) => {
```

Comment:
Do we need both the `await Promise.all(` and the `await topicNames.map` awaits? Intuitively it feels like the outer await isn't going to be doing anything, but I'm not sure.

Reply:
Yeah you are right. Removed the outer await.
PTAL!